src/modules/NewDream/NewDream.js

```diff
@@ -120,6 +121,22 @@ const NewDream = (props) => {
               className: classes.text,
             }}
           ></TextField>
+          <TextField
+            id='dream-emotion'
+            variant='outlined'
+            color='primary'
+            label='Emotion of Dream'
+            fullWidth
+            multiline
+            rowsMax={1}
```

Comment:
Can remove both of these, I'm fairly certain the default TextField behavior is rowsMax=1, so if you remove the multiline prop it should behave as expected

Reply:
Great catch! Removed, thank you